src/controllers/messages.controller.ts

```diff
@@ -0,0 +1,208 @@
+import { messagesDB } from '../models/db.index';
+import { RequestHandler } from 'express';
+import createHttpError from 'http-errors';
+
+export const getMessages: RequestHandler = async (req, res, errFunc) => {
+    try {
+        const messages = await messagesDB.findAll();
+        res.status(200).json(messages);
+    } catch (error) {
+        errFunc(error);
+    }
+};
+export const getLastMessages: RequestHandler = async (req, res, errFunc) => {
+    try {
+        const numMsgs = (req.params.numMsgs as unknown as number) || 10; //cast to number, or default to 10
```

Comment:
can we change this to req.query? otherwise default is 10. will need to update the route address